next/pages/all-articles/[[...page]].tsx

```diff
@@ -85,6 +90,11 @@ export const getStaticProps: GetStaticProps<AllArticlesPageProps> = async (
       : prevEnabled && [pageRoot, prevPage].join("/");
   const nextPageHref = nextEnabled && [pageRoot, nextPage].join("/");
 
+  // Create language links for this page.
+  // Note: the links will always point to the first page, because we cannot guarantee that
+  // the other pages will exist in all languages.
+  const languageLinks = createLanguageLinksForNextOnlyPage(pageRoot, context);
```

Comment:
the `pageRoot` constant is already defined at line 84, and is used for the pagination links, so we can reuse it here.

(also note the comment, the language link will take users to the first page of the listing, which is a good feature I think).